core/src/main/java/hudson/tools/DownloadFromUrlInstaller.java

```diff
@@ -148,6 +146,23 @@ public String getId() {
             if(d==null)     return Collections.emptyList();
             return Arrays.asList(((InstallableList)JSONObject.toBean(d,InstallableList.class)).list);
         }
+        
+        /**
+         * The UI uses this information to populate the drop-down.
+         * 
+         * @return JSON array of installable tools, never null
+         * @since 1.617
+         */
+        public JSONArray getInstallablesJson() throws IOException {
```

Comment:
Could you elaborate, which it should be restricted? I don't see any issue with providing access to versions and URLs downloaded by back end crawlers and left this annotation intentionally. Thanks.